packages/components/src/systems/PatientMedHistory/index.tsx

```diff
@@ -42,6 +43,25 @@ type PatientMedHistoryProps = {
   newMedication?: Medication | SearchMedication;
 };
 
+type PatientMedication = {
+  active: boolean;
+  prescription: {
+    id: string;
+    writtenAt: string;
+  };
+  medication: {
+    id: string;
+    name: string;
+  };
+};
+
+type GetPatientResponse = {
+  patient: {
+    id: string;
+    medicationHistory: PatientMedication[];
```

Comment:
Its a little bit of a nit, but the schema says this is optional. Idk what we actually return for a patient with no medication history. https://github.com/Photon-Health/terraform/blob/main/modules/graphql-api/appsync.graphql#L108